content/doc/book/installing/index.adoc

```diff
@@ -1,79 +1,334 @@
+  (JDK) is fine +
+  *Note:* This is not a requirement if running Jenkins as a <<docker,Docker>>
+  container.
 
-=== Experimentation, Staging, or Production?
 
-How you configure Jenkins will differ significantly depending on your intended use cases.
-This section is specifically targeted to initial use and experimentation.
-For other scenarios, see link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation].
+== Installation platforms
 
-=== Stand-alone or Servlet?
+This section describes how to install/run Jenkins on different platforms and
+operating systems.
 
-Jenkins can run stand-alone in its own process using its own built-in web server (Jetty).
-It can also run as one servlet in an existing framework, such as Tomcat or Glassfish application servers.
-This section is specifically targeted to stand-alone install and execution.
-For other scenarios, see link:/doc/book/appendix/advanced-installation/[Advanced Jenkins Installation]
 
-== Installation
+=== Docker
+
+As described on the https://docs.docker.com/engine/docker-overview/[Docker
+overview] page, Docker is a platform for running applications in an isolated
+environment called a "container" (or Docker container). Applications like
+Jenkins can be downloaded as read-only "images" (or Docker images), each of
+which is run in Docker as a container. Therefore, a Docker container is in
+effect a "running instance" of a Docker image. From this perspective, an image
+is stored permanently more or less (i.e. insofar as image updates are
+published), whereas containers are stored temporarily. Read more about these
+concepts in the Docker documentation's
+https://docs.docker.com/get-started/[Getting Started, Part 1: Orientation and
+setup] page.
+
+Docker's fundamental platform and container design means that a single Docker
+image (for any given application like Jenkins) can be run on any operating
+system (e.g. macOS, Linux and Windows) which is also running Docker.
+
+
+==== Installing Docker
+
+To install Docker on your operating system, visit the
+link:https://www.docker.com/[Docker] website and navigate to *Get Docker* at the
+top of their site to begin accessing the Docker download that's suitable for
+your operating system. Follow the installation instructions on their website.
+
+*Note:* If you are installing Docker on a Linux-based operating system, ensure
+you configure Docker so it can be managed as a non-root user. Read more about
+this in Docker's
+https://docs.docker.com/engine/installation/linux/linux-postinstall/[Post-installation
+steps for Linux] page of their documentation. This page also contains
+information about how to configure Docker to start on boot.
+
+
+==== Downloading and running Jenkins in Docker
+
+There are several Docker images of Jenkins available.
+
+However, the recommended Docker image to use is the
+link:https://hub.docker.com/r/jenkinsci/blueocean/[`jenkinsci/blueocean` image].
```

Comment:
Discussion for styleguide "link:" or no "link:"
https://issues.jenkins-ci.org/browse/WEBSITE-268

Reply:
Sorry I didn't see WEBSITE-268, which I've now linked to [WEBSITE-432](https://issues.jenkins-ci.org/browse/WEBSITE-432).